sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/BackupRestoreRestClient.cs

```diff
@@ -63,7 +63,7 @@ internal HttpMessage CreateFullBackupRequest(string vaultBaseUrl, SASTokenParame
         /// <param name="vaultBaseUrl"> The vault name, for example https://myvault.vault.azure.net. </param>
         /// <param name="azureStorageBlobContainerUri"> Azure blob shared access signature token pointing to a valid Azure blob container where full backup needs to be stored. This token needs to be valid for at least next 24 hours from the time of making this call. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public async Task<ResponseWithHeaders<FullBackupDetailsInternal, ServiceFullBackupHeaders>> FullBackupAsync(string vaultBaseUrl, SASTokenParameter azureStorageBlobContainerUri = null, CancellationToken cancellationToken = default)
```

Comment:
Is this response model change expected as a result of [adding "x-ms-long-running-operation": true to the operation](https://github.com/Azure/azure-rest-api-specs/pull/9898/files#r444498598) ?